src/jvm/main/org/jetbrains/kotlinx/lincheck/runner/FixedActiveThreadsExecutor.kt

```diff
@@ -213,7 +213,12 @@ internal class FixedActiveThreadsExecutor(private val testName: String, private
 
 }
 
-private val majorJavaVersion = Runtime.version().version()[0]
+private val majorJavaVersion = runCatching {
```

Comment:
Would `System.getProperty("java.specification.version")` work as a replacement?

Reply:
Do you mean providing it explicitly as an argument?

Reply:
As far as I understand, it should be provided automatically by the JVM.